frontend/package.json

```diff
@@ -172,9 +168,9 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^1.7",
     "surge": "^0.23.1",
-    "typescript": "^3.9.7"
+    "typescript": "4.4.4"
```

Comment:
should probably be ^...

Reply:
Let's open an issue for upgrading directly to latest version, which now is 5.3.3, since upgrading turns out it requires some changes